.npmignore

```diff
@@ -0,0 +1,16 @@
+.appveyor.yml
+.editorconfig
+.eslintrc
+.github
+.gitignore
+.npmignore
+.sasslintrc
+.travis.yml
+backstop.conf.js
+bower.json
+changelog.md
+composer.json
+package.js
+package.json
+src
```

Comment:
@styfle which your opinion about remove the source code of the request package?

Reply:
You don’t need src, only dist

Reply:
This is the same file from the previous commit. I’m fixing the regression.